javascript/javascript1/week4/Voice-assistant.js

```diff
@@ -0,0 +1,117 @@
+let name = ""
+const todoList = []
+
+function getReply(command) {
+ if (typeof command !== "string" || command === "") {
+   return "Please enter a valid command"
+ }
+/* let num1 = x;
+let num2 = y;*/
+ switch(command){
+
+   case "Hello my name is Benjamin":
+     name = "Benjamin"
+       return `Nice to meet you Benjamin`
+
+   case "What is my name?":
+       if(name !== ""){
```

Comment:
I think you should check if the name `IS` an empty string and then return "You didn't say your name yet" or 
```suggestion
       if(!name){
```